Casks/p/progressive-downloader.rb

```diff
@@ -2,14 +2,14 @@
   version "6.4"
   sha256 "fe33f7246f4476db654060b160d3cc43e32afcb8f1432f239281187adda96580"
 
-  url "https://www.macpsd.net/update/#{version}/PSD.dmg"
+  url "https://www.macpsd.net/update/#{version}/PSD_#{version.dots_to_underscores}.dmg"
   name "Progressive Downloader"
   desc "Download manager"
   homepage "https://www.macpsd.net/"
 
   livecheck do
     url :homepage
-    regex(%r{href=.*?/(\d+(?:\.\d+)+)/PSD\.dmg}i)
+    regex(%r{href=.*?/(\d+(?:\.\d+)+)/PSD[._-]?.+\.dmg}i)
```

Comment:
```suggestion
    regex(%r{href=.*?/(\d+(?:\.\d+)+)/PSD[^"' >]*?\.dmg}i)
```

If we want the filename version to be optional (matching both `PSD.dmg` and `PSD_6_4.dmg`), `[^"' >]*?` would be more appropriate within the context of an `href` attribute (i.e., it's non-greedy and would stay within the bounds of the attribute) and we can omit the `[._-]?` if it's also optional.

Reply:
Thank you 👍🏻